config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1580,3 +1580,52 @@ presubmits:
           limits:
             cpu: 4
             memory: 6Gi
+
+  - name: pull-kubernetes-e2e-inplace-pod-resize-containerd-main
+    cluster: k8s-infra-prow-build
+    always_run: false
+    optional: true
+    skip_report: false
+    run_if_changed: '^(pkg\/kubelet\/kuberuntime\/|test\/e2e\/node/)'
+    skip_branches:
+      - release-\d+\.\d+  # per-release image
+    annotations:
+      testgrid-dashboards: sig-node-presubmits
+      testgrid-tab-name: pr-kubelet-gce-cluster-e2e-inplace-pod-resize-containerd-main
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221107-33c989e684-master
+        args:
+          - --repo=github.com/containerd/containerd=main
```

Comment:
you need to add 


```
KUBE_MASTER_EXTRA_METADATA / KUBE_NODE_EXTRA_METADATA
```

since we are using containerd main branch

see https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-node/containerd.yaml#L1068-L1070